src/sentry/static/sentry/app/views/substances/substances.jsx

```diff
@@ -36,7 +45,7 @@ class Substances extends React.Component {
       {
         Header: 'Container',
         id: 'container',
-        accessor: d => d.properties.container,
+        accessor: d => d.position.container.name,
```

Comment:
I guess this is related to the hard coded position property from the substance endpoint? I'm wondering when we move on with this if this should be property like it was before, or if it should be tagged directly onto the substance object?

Reply:
I think that we had it in the properties as a hack to show it. I would prefer that we would have it on the object.